packages.json

```diff
@@ -22188,5 +22188,19 @@
     "description": "create db util sources from csv",
     "license": "MIT",
     "web": "https://github.com/z-kk/csv2dbsrc"
+  },
+  {
+    "name": "nptr",
+    "url": "https://15b1acb3d21ebde45e8ab271e72fb4b75ad40143:x-oauth-basic@github.com/henryas/nptr",
```

Comment:
Why not https://github.com/henryas/nptr ?

Reply:
It was autogenerated by nimble. It should be https://github.com/henryas/nptr

Reply:
This is my first nimble package and I am new to this. So do let me know if there is anything else I need to do. Thank you.

Reply:
You can edit the link directly.